src/lexer/tokens.rs

```diff
@@ -12,126 +12,126 @@ pub enum Token {
     #[token("@EXTERNAL")]
     PropertyExternal,
 
-    #[token("PROGRAM", ignore(case))]
+    #[token("PROGRAM")]
```

Comment:
Could you revert this?
I suggest we add the lexer to a separate crate later to avoid the issues